src/main/java/io/ebean/EbeanVersion.java

```diff
@@ -13,31 +13,31 @@
  */
 public class EbeanVersion {
   private EbeanVersion() {
-    
+
   }
   private static final Logger logger = LoggerFactory.getLogger(EbeanVersion.class);
-  
+
   private static String version = "unknown";
   static {
     try {
       Properties prop = new Properties();
-      InputStream in = Ebean.class.getResourceAsStream("/META-INF/maven/io.ebean/ebean/pom.properties");
-      if (in != null) {
-        prop.load(in);
```

Comment:
OLD: if prop.load throws an exception, the stream was not closed
NEW: try-with-resources will ensure, that the stream is closed properly